lumber-generate.js

```diff
@@ -36,15 +35,13 @@ program
 
   let schema = {};
 
-  if (program.db) {
```

Comment:
Is `db: program.db,` still useful in the General Prompter (line 15)?

Reply:
Missed it, thanks :)